src/en/rinkocomics/build.gradle

```
@@ -0,0 +1,7 @@
+ext {
+    extName = 'Rinko Comics'
+    extClass = '.RinkoComics'
+    extVersionCode = 1
```

Should be set explicitly.
```suggestion
    extVersionCode = 1
    isNsfw = false
```